jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -595,7 +599,7 @@ YUI.add('juju-env-api', function(Y) {
       var cback = function(err, response) {
         this.handleLogin({error: err, response: response});
         if (callback) {
-          callback(err);
+          callback(err, response);
```

Comment:
Good catch, I updated the docstring - I found it very helpful to debug the error case when the response was passed through.